src/v0/destinations/ga4/transform.js

```diff
@@ -428,8 +428,12 @@ const process = (event) => {
       break;
     case EventType.PAGE:
```

Comment:
Do we still need to handle this case when we already filter it with allowed message types in the definition config?

Reply:
yes, @saikumarrs because there are customers who are using ga4 cloud mode page calls so we can't remove that support. right now this is a temporary solution but we need to come up with permanent solution which is server should know if hybrid mode is enabled then for a destination which calls should pass through cloud mode and which not

Reply:
Also can you check above comment by @krishna2020 can we do something like that?

Reply:
Yeah, let's proceed.